src/features/home/components/modules/video/VideoMultiOfferTile.native.test.tsx

```diff
@@ -34,6 +34,10 @@ jest.mock('@batch.com/react-native-plugin', () =>
   jest.requireActual('__mocks__/libs/react-native-batch')
 )
 
+const user = userEvent.setup()
+
+jest.useFakeTimers()
```

Comment:
Pourquoi on ajoute des fake timers ? C'est suite au userEvent ? Ou c'est décarrelé ?

Reply:
Yes c'est nécessaire sinon on a une erreur, on a mis à jour la doc par [ici](https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/development/tests/unit-test/general-guidelines.md#when-to-use-web-and-native)